pages/community/precice-workshop-2024.md

```diff
@@ -51,18 +57,105 @@ The cost of lunch, as well as coffee and snacks is included in the registration
 - 12:00-13:30: 🍲 Registration and lunch
 - 13:30-15:00: Official welcome + user introductions
 - 15:00-15:30: ☕️ Coffee break
-- 15:30-17:00: What's new in preCICE?
+- 15:30-17:00: What's new in preCICE?<br/>
+  <details class="workshop-event" id="talk-v3">
+    <summary>
+      Overview on preCICE v3<br/>
+      <p><a href="https://www.ipvs.uni-stuttgart.de/institute/team/Simonis/">Frédéric Simonis</a> (<a href="https://github.com/fsimonis">@fsimonis</a>), University of Stuttgart, Germany</p>
+    </summary>
+    <p>Version 3 of the preCICE library has arrived and brings a wide range of changes, improvements, and additional features. This talk aims to give a general overview of what has changed and what is being worked on.</p>
```

Comment:
This is now a slot split between Frédéric and me, where Frédéric talks about the library and I talk about the ecosystem. I have submitted an abstract about the ecosystem talk.